deps/build.jl

```diff
@@ -59,6 +59,7 @@ end
 
 # Windows
 if is_windows()
+    rm(joinpath(dirname(@__FILE__), "usr"); force = true, recursive = true)
```

Comment:
though you'd want to escape backslashes in windows paths to match how the user would need to enter them, which might not happen automatically